en/ros2/user_guide.md

```diff
@@ -768,43 +804,62 @@ average rate: 247.485
 The `ros2 launch` command is used to start a ROS 2 launch file.
 For example, above we used `ros2 launch px4_ros_com sensor_combined_listener.launch.py` to start the listener example.
 
-You don't need to have a launch file, but they are very useful if you have a complex ROS 2 system that needs to start several components. 
+You don't need to have a launch file, but they are very useful if you have a complex ROS 2 system that needs to start several components.
 
 For information about launch files see [ROS 2 Tutorials > Creating launch files](https://docs.ros.org/en/humble/Tutorials/Intermediate/Launch/Creating-Launch-Files.html)
 
-
-
 ## Troubleshooting
 
 ### Missing dependencies
 
 The standard installation should include all the tools needed by ROS 2.
 
 If any are missing, they can be added separately:
+
 - **`colcon`** build tools should be in the development tools.
   It can be installed using:
+
   ```sh
   sudo apt install python3-colcon-common-extensions
   ```
+
 - The Eigen3 library used by the transforms library should be in the both the desktop and base packages.
   It should be installed as shown:
 
-   :::: tabs
+  :::: tabs
 
-   ::: tab humble
-   ```sh
-   sudo apt install ros-humble-eigen3-cmake-module
-   ```
-   :::
-   
-   ::: tab foxy
-   ```sh
-   sudo apt install ros-foxy-eigen3-cmake-module
-   ```
-   :::
-   
-   ::::
+  ::: tab humble
+
+  ```sh
+  sudo apt install ros-humble-eigen3-cmake-module
+  ```
+
+  :::
+
+  ::: tab foxy
+
+  ```sh
+  sudo apt install ros-foxy-eigen3-cmake-module
+  ```
+
+  :::
 
+  ::::
+
+### ros_gz_bridge not publishing on the \clock topic
```

Comment:
This section below is the warning, turned into a section. Can you sanity check that too please. IN particular the instruction to remove the wrong version, and the line that starts "This might happen ...".

Reply:
For me, It is good. I have checked and when someone installs `ros-humble-ros-gzgarden` in automatic way it also installs  the correct bridge